### PR TITLE
OSDOCS-2975: Add supported profiles table

### PR DESCRIPTION
**Pull request by kmccarron-rh**
Applies to 4.6+.
Jira Card: https://issues.redhat.com/browse/OSDOCS-2975

Compliance Operator release notes are updated to include the profiles that were added in release 0.1.44 (see the last 2 bullets in this section):
https://deploy-preview-39324--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-release-notes.html#compliance-operator-0-1-44-new-features-and-enhancements

A table listing the supported profiles is added here:
https://deploy-preview-39324--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-supported-profiles.html#compliance-supported-profiles_compliance-operator-supported-profiles

### CODE REVIEW

File: modules/compliance-supported-profiles.adoc [hunk 1]
```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/
+
+[id="compliance-supported-profiles_{context}"]
+= Compliance profiles
+
+The Compliance Operator provides the following compliance profiles:
+
+.Supported compliance profiles
```
Comment (rhmdnd) [inline modules/compliance-supported-profiles.adoc:10]: This table renders nicely. Are we going to add something that tells people how to find this information in the deployment itself? After reading the Jira ticket referenced in the commit message, it sounded like we wanted to provide some sort of statement describing how users can find this information themselves. Is my understanding correct that that information comes a few sections later?

https://deploy-preview-39324--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-understanding.html#compliance_profiles_understanding-compliance

Comment (kmccarron-rh) [reply modules/compliance-supported-profiles.adoc:10]: Hi @rhmdnd - The original request was to add this to Understanding section, but during a meeting, we decided to try it up a level in the doc. If the current positioning in the book is good,  I can add a link over to the Understanding section that explains how to find the profiles on your system.  Does that seem like a good approach? Alternately, I could just move the supported profile section under the Understanding  section, but I'm afraid that it gets obscured there.

Comment (rhmdnd) [reply modules/compliance-supported-profiles.adoc:10]: I'll ultimately defer to others, but I think the current positioning looks great. As a reader, I like having a table with the supported profiles up front. Your suggestion about adding a link to the Understanding section makes sense.

**Comment by rhmdnd** [comment]
+1

**Comment by kmccarron-rh** [comment]
This one is ready for review.
SME:
@Vincent056 
@nkinder 
QE:
@pdhamdhe  for QE review

**Review by Vincent056 — COMMENTED** [review]
I think the table and this PR look very good! Sorry didn't see it earlier, I noticed something not relevant to this PR when I was browsing the rendered page in https://deploy-preview-39324--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-release-notes.html#templating-and-variable-use section, where stats " Additionally, the ComplianceCheckResult objects now use the label compliance.openshift.io/check-has-value that lists the variables a check can use.", I think it should be "lists the variables a check has used." instead, as the values only appear when used by the check.

**Review by Vincent056 — APPROVED** [review]
/lgtm

**Comment by kmccarron-rh** [comment]
Adding the change that Vincent noted above.

**Comment by Vincent056** [comment]
> Adding the change that Vincent noted above. Also, removing the exclusive language statement per conversation in the Team-docs channel. This is covered in the main OCP release notes.

Thanks for that!

**Review by lpettyjo — COMMENTED** [review]
/LGTM

**Pull request merged**